draft-ietf-mops-streaming-opcons.md

```diff
@@ -33,6 +33,12 @@ author:
     org: Tencent America LLC
     country: United States of America
     email: spencerdawkins.ietf@gmail.com
+ -
+    ins: M. Stock
```

Comment:
Has this been discussed/resolved yet?

https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/pull/18#discussion_r453389046

Reply:
Not much or very specifically, but I'll give my own take:
There was a point when it was probably open to add another author that would take on a range of new issues and the various tasks involved in getting this doc to a publishable state, but that's probably a bit late by now, assuming nothing goes too terribly wrong.  The PR has a significant contribution for which I'm grateful, but I don't think it's quite the same as going into the author's list as it stands.

But with that said, it's definitely true that a few people have contributed at a significantly higher level than some of the others, and this is a good candidate for one of them--it tends to lean toward "above and beyond" to post several paragraphs of proposed draft text, as opposed to hand-waving at a general point for the authors to address from the mic line.  Although that latter is also a welcome kind of feedback, in some cases it's harder to turn into a value-add to the doc than proposed text is.

I guess the relevant advice is Sections 4.10 and 4.11 from RFC 7322, and they seem to leave a wide latitude on how exactly to do it:
https://datatracker.ietf.org/doc/html/rfc7322#section-4.10

I haven't seen many RFCs that differentiate by level of effort in the Acknowledgements section or add a Contributor's section, but if a highlighted call-out of some sort would be meaningful there, we could try discussing the best way to put such a thing?  Or maybe you and he would rather have an offer of a beer with a personal thank-you next time we're in the same area, and he can go in the regular list of acknowledgements instead of the authors?  I'm basically aiming to land on nobody feeling slighted here while taking all the contributions that improve the doc.

But I think that summary is "let's drop these particular lines, and open a new issue to get the acknowledgements right if that's needed".